### PR TITLE
cleanup mock data

### LINKED ISSUE #922: Clean up mock data and refactor components to use proper data types [Atlas]
Right now we use naive mock data (strings for view counts, formatted string with dates, etc.). Also, our mock data is rather repetitive and lots of images are broken.

We should improve the mock data by adding some different scenarios and make the components work with proper types/objects.

### PR DESCRIPTION
**Pull request by kdembler**
This PR cleans up mocked data used by Atlas by:

* adding set of images to use
* adding varied data for both videos and channels so we can observe how the app behaves with different data length, etc.
* cleaning up the home view to correspond to match what we agreed upon in https://github.com/Joystream/joystream/issues/824#issuecomment-653048676

Resolves #922

### CODE REVIEW

File: atlas/packages/app/src/utils/number.ts [hunk 1]
```diff
@@ -1,3 +1,18 @@
 export const formatNumber = (num: number): string => {
   return num.toLocaleString('en-US').split(',').join(' ')
 }
+
+export const formatNumberShort = (num: number): string => {
```
Comment (fcescob) [inline atlas/packages/app/src/utils/number.ts:5]: There is a tiny inconsistency here, this returns `3.0K` for `3001` and `3K` for `3000`, same things happens with milions

**Review by fcescob — COMMENTED** [review]
Beside one minor thing, looks good to me. 

Atlas looks stunning with images!
immagine

File: atlas/packages/app/src/utils/number.ts (hunk 1 above)
Comment (kdembler) [reply atlas/packages/app/src/utils/number.ts:5]: Good catch! Changed the function to always skip `.0` suffix

File: atlas/packages/app/src/utils/number.ts [hunk 2]
```diff
@@ -14,5 +14,10 @@ export const formatNumberShort = (num: number): string => {
     suffix = 'K'
   }
 
-  return `${value % 1 === 0 ? value : value.toFixed(1)}${suffix}`
+  let formattedValue = value.toFixed(1)
+  if (formattedValue.endsWith('.0')) {
+    formattedValue = formattedValue.slice(0, formattedValue.length - 2)
```
Comment (fcescob) [inline atlas/packages/app/src/utils/number.ts:19]: Formatting is still a bit inconsistent,  `1000` and `1001` will get two different representations, `1000` and `1K` respectively, same happens for `1000000` and `1000001`


```javascript
// This evaluates to "1000" 
formatNumber(1000)
// This evaluates to "1K"
formatNumber(1001)
```

Comment (kdembler) [reply atlas/packages/app/src/utils/number.ts:19]: ofc, that should have been `>=` instead of `>`. Fixed now!

**Review by fcescob — APPROVED** [review]
Good job!

**Pull request merged**